readme.md

```diff
@@ -1,6 +1,32 @@
-# Boilerplate for layout tasks
+# Frontend practice with header
 
-Here is the description.
+## Main goal:
+Create HTML page with header. Understand `flexbox` usage. Create
+HTML layout from the mockup.
+
+Use this [mockup](#) for development.
```

Comment:
Do we have the mock? it may be different from out reference implementation.

Reply:
@roman-mate I'm waiting for mock from Alesya. She has done a general mockup and now we wait for splitting it to different tasks

Reply:
asesome.